autoload/vsnip/variable.vim

```diff
@@ -73,6 +73,11 @@ function! s:TM_FILEPATH(context) abort
 endfunction
 call vsnip#variable#register('TM_FILEPATH', function('s:TM_FILEPATH'))
 
+function! s:RELATIVE_FILEPATH(context) abort
+  return expand('%')
```

Comment:
I think `:~:.` might be better here (in the Vim docs it says this):
```
	:.	Reduce file name to be relative to current directory, if
		possible.  File name is unmodified if it is not below the
		current directory.
		For maximum shortness, use ":~:.".

```
Not totally sure in which case this will lead to a difference though...

Reply:
Yes good point. I guess I was primarily concerned with files under the CWD as you pretty unlikely to want absolute paths expanded in your source code, so it's pretty edge case. However, I had a quick look at how VS Code works for `$RELATIVE_FILEPATH` for files outside of CWD and expands them as absolute paths.

Vim is a little inconsistent with `expand('%')` in that it renders (most of the time) the path you used when loading the file, that is, the filename you used with `:e`, which might be relative or absolute.

Arguably we should make this behave the same as VS Code, but also it seems like relative paths (even if outside of the CWD) might be useful in some cases. As I said this is already an edgecase so I'm happy to leave it until somebody finds a good reason to change it. WDYT?

Reply:
Good points, I tested it a bit and couldn't really find a big difference between the two options. So let's leave it like this for now :+1: